### ISSUE TITLE
RFC: 3.0 - Helper aliasing

### ISSUE THREAD

**Issue by rchavik** [croogo/croogo#708]
Views should not be using `CroogoForm` or `CroogoHtml` helper.  Those helpers should aliased as `Form` and `Html` respectively, as called via `$this->Form` or `$this->Html` in views.

These helpers/aliases can be overriden by themes, ie: they can load their own custom helpers but still called by the default `Form` and `Html` name.

It might be worth considering to refactor out Croogo specific methods to a separate class:

- [ ] `Html::icon()`
- [ ] `Html::beginBox()`
- [ ] `Html::endBox()`
- [ ] `Html::status()`
- [ ] `Html::tabStart()`
- [ ] `Html::tabEnd()`
- [ ] `Form::autocomplete()`, but this might not be trivial as it may require internal state from the instance created by `Form::create()`.

**Comment by Marlinc** [comment]
I'd personally like to move Croogo specific like the admin tabs and autocomplete in a separate plugin. That way a theme creator can simply extend the CakePHP helpers. I think it would also take away a bit of confusion.

**Comment by Wouter0100** [comment]
I agree with Marlin here, I don't think it's a good idea to merge theme specific things in Croogo Core (only things like the URL helper/selector).
